.evergreen/check-rustfmt.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/sh
+#!/bin/bash
 
 set -o errexit
 
 . ~/.cargo/env
-cargo +nightly fmt -- --check
+rustfmt +nightly --unstable-features --check src/**/*.rs
```

Comment:
do we need to keep using the nightly version if we're using `unstable-features`?

Reply:
`--unstable-features` is a nightly-only flag